assets/translations/en.json

```diff
@@ -285,7 +284,8 @@
     "normal": "Normal",
     "shizuku": "Shizuku",
     "root": "Root",
-    "shizukuBinderNotFound": "Shizuku is not running",
+    "shizukuBinderNotFound": "Compatible Shizuku service wasn't found",
+    "installMethodsExplanation": "Alternative installation methods allow you to bypass the limitations of Android. This can, for example, solve problems when using firmware like HyperOS(MIUI).\nNormal or Shizuku is recommended.",
```

Comment:
I don't think we need this. Someone who needs Shizuku or root probably already knows what that is. Besides it's very long - might make more sense to add this to the wiki instead of the app itself.